docs/guides/audit/seccomp.md

```diff
@@ -41,6 +41,9 @@ spec:
     - mkdir
 ```
 
+Note that, on `arm64` based computer, you will need to replace
+`SCMP_ARCH_X86_64` by `SCMP_ARCH_AARCH64`.
```

Comment:
The `architectures` list is optional according to runtime-spec:
https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md#seccomp

Man seccomp_arch_add.3:
> When a seccomp filter is initialized with the call to seccomp_init(3) the native architecture is automatically added to the filter.

And runc only calls AddArch but never RemoveArch:
https://github.com/opencontainers/runc/blob/4ff59f81fded69a6d5f9e85d7eda5b23587db985/libcontainer/seccomp/seccomp_linux.go#L79-L88

So we could remove the `architectures` section from the yaml for simplicity.

Some of the examples from SPO don't have `architectures`:
https://github.com/kubernetes-sigs/security-profiles-operator/blob/de6b5a05bafbf86c65b6ef5b772d77ba4d6a72d0/examples/seccompprofile-allowed-syscalls-validation.yaml#L42-L49